### PR TITLE
feat: add lsp (liqudity-slicing ) protocol fees data

### PR DESCRIPTION
**Pull request by libaice**
In this PR , we added the daily fees statistics of LSP protocol (https://www.lsp.network/)
(http://github.com/defiLlama/DefiLlama-Adapters/blob/main/projects/liquidity-slicing)

Methodology
1. calculate the reserve wallet daily USDT balance
2. get the totalUSD reward from our user positon API
3. sum the usd balance of these two values .

### CODE REVIEW

**Comment by llamabutler** [comment]
The liquidity-slicing adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees liquidity-slicing
    
    🦙 Running LIQUIDITY-SLICING adapter 🦙
    ---------------------------------------------------
    Start Date:	Wed, 11 Dec 2024 10:32:54 GMT
    End Date:	Thu, 12 Dec 2024 10:32:54 GMT
    ---------------------------------------------------
    
    ARBITRUM 👇
    Backfill start time: 18/11/2024
    Methodology: LSP charges a 0.5% fee (in USDT) on market transactions, and takes 10% fee on users staking rewards.
    Daily fees: 1.49 k
    End timestamp: 1733999573 (2024-12-12T10:32:53.000Z)

File: fees/liquidity-slicing/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    });
+    
+    const apiResponse = await httpPost(API_ENDPOINT, {
+        user_address: FEE_RECEIVER_ADDRESS
+    });
+    
+    const totalStakeUsd = apiResponse.result.reduce((sum: number, pool: PoolData) => 
+        sum + Number(pool.stake_amount_usd), 0);
+    
+    dailyFees.add(ARB_USDT, balance);
```
Comment (dtmkeng) [inline fees/liquidity-slicing/index.ts]: balance it is total not daily

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by libaice** [comment]
Yeah , Let me change this calcultion

**Comment by llamabutler** [comment]
The liquidity-slicing adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees liquidity-slicing
    
    🦙 Running LIQUIDITY-SLICING adapter 🦙
    ---------------------------------------------------
    Start Date:	Mon, 23 Dec 2024 06:33:06 GMT
    End Date:	Tue, 24 Dec 2024 06:33:06 GMT
    ---------------------------------------------------
    
    ARBITRUM 👇
    Backfill start time: 18/11/2024
    Methodology: LSP charges a 0.5% fee (in USDT) on market transactions, and takes 10% fee on users staking rewards.
    Daily fees: 1.17 k
    End timestamp: 1735021985 (2024-12-24T06:33:05.000Z)

**Comment by llamabutler** [comment]
The liquidity-slicing adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees liquidity-slicing
    
    🦙 Running LIQUIDITY-SLICING adapter 🦙
    ---------------------------------------------------
    Start Date:	Mon, 23 Dec 2024 06:39:38 GMT
    End Date:	Tue, 24 Dec 2024 06:39:38 GMT
    ---------------------------------------------------
    
    ARBITRUM 👇
    Backfill start time: 18/11/2024
    Methodology: LSP charges a 0.5% fee (in USDT) on market transactions, and takes 10% fee on users staking rewards.
    Daily fees: 31
    End timestamp: 1735022377 (2024-12-24T06:39:37.000Z)

**Comment by libaice** [comment]
Hello @dtmkeng , I have add new API to calculate the liquidity slicing protocol daily fees amount, pls help me merge this Pr . 

Thx 💗

**Pull request merged**